### PR TITLE
{Core} Update Next Breaking Change Release Version Number

### PR DESCRIPTION
**Pull request by ReaNAiveD**
**Related command**

**Description**

Update the breaking change release version.
It is recommended to update the version during the breaking change window. This will help avoid any issues with pre-announcements that indicate changes will occur in the current version after relase.

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

File: src/azure-cli-core/azure/cli/core/breaking_change.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 logger = get_logger()
 
-NEXT_BREAKING_CHANGE_RELEASE = '2.67.0'
+NEXT_BREAKING_CHANGE_RELEASE = '2.73.0'
```
Comment (bebound) [inline src/azure-cli-core/azure/cli/core/breaking_change.py:16]: We use `2.74` in this PR.
Please double confirm the version number.

Comment (ReaNAiveD) [reply src/azure-cli-core/azure/cli/core/breaking_change.py:16]: I reached out to Yong regarding the next breaking change version number and received `2.73.0` as the response. I'm flexible with either `2.73.0` or `2.74.0`, as I plan to update to the confirmed version as soon as it's finalized.

Comment (bebound) [reply src/azure-cli-core/azure/cli/core/breaking_change.py:16]: Find `2.73.0` in https://learn.microsoft.com/en-us/cli/azure/azure-cli-support-lifecycle too.
Hmm, #30091 might be wrong.

Comment (bebound) [reply src/azure-cli-core/azure/cli/core/breaking_change.py:16]: `2.54.0` is 2023 ignite version, `2.61.0` is 2024 build version. Weird.

Comment (jiasli) [reply src/azure-cli-core/azure/cli/core/breaking_change.py:16]: I got 2.74 from @dcaro: https://github.com/Azure/azure-cli/pull/30091#discussion_r1819649786.

Comment (ReaNAiveD) [reply src/azure-cli-core/azure/cli/core/breaking_change.py:16]: I think it's acceptable to merge this PR for now. We can create another PR once we have the finalized version.

**Pull request merged**